pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -692,13 +691,7 @@ func (s *DiscoveryServer) pushEds(push *model.PushContext, con *XdsConnection, v
 			}
 		}
 
-		var l *xdsapi.ClusterLoadAssignment
-		// decide which to use based on presence of Sidecar.
-		if sidecarScope == nil || sidecarScope.Config == nil || len(features.DisableEDSIsolation) != 0 {
```

Comment:
So we're also removing the 'disable EDS isolation' ? I guess it's ok.

Reply:
I guess this doesn't need to be in this change. This change was broken off the larger change of completely removing the legacy EDS code, but that turned out to be a lot more complicated. Should be no problems keeping it if we still want it, although it seems safe to remove now